### ISSUE TITLE
Wifi Load

### ISSUE THREAD

**Issue by ShadowJonathan** [pdreker/fritz_exporter#323]
With a FRITZ!Box 7590, I'm able to see the load on the current wifi channel (plus other sources), and has been in the past very useful in detecting wifi issues:
<img width="1282" alt="image" src="https://github.com/pdreker/fritz_exporter/assets/22740616/7df7324b-296e-4379-99df-72fd42b03e34">

Could this data be scraped by this exporter? I'm not sure if it's exposed, but I also haven't been able to find an issue on here that mentions it.

**Comment by pdreker** [comment]
Sorry for not replying for... oh man, more than half a year, but somehow this dropped off my radar.

This exporter basically uses the fritzconnection library to extract data from the box, so anything not exposed by that library will not be available. The problem goes a bit deeper than that though: fritzconnection uses only documented interfaces to the boxes, as this keeps the complications withion bounds. So anything not available via TR-064 or fritz_AHA (the Homeautomation Part) will not be available from this exporter.

Long story short: I would like to provide more data, but unless I can get it, without querying undocumented endpoints on the boxes, chances are low.
